lib/ftp/doc/archive/rfc959.txt

```diff
@@ -78,7 +78,7 @@ File Transfer Protocol
       RFC 354 obsoleted RFCs 264 and 265.  The File Transfer Protocol
       was now defined as a protocol for file transfer between HOSTs on
       the ARPANET, with the primary function of FTP defined as
-      transfering files efficiently and reliably among hosts and
+      transferring files efficiently and reliably among hosts and
```

Comment:
The RFC is a copy of the upstream RFC, so leave that text as it is. (Yes, the upstream RFC has spelling mistakes)

Reply:
> The RFC is a copy of the upstream RFC, so leave that text as it is. 

@garazdawi Fixed.

> (Yes, the upstream RFC has spelling mistakes)

:rofl:

Reply:
You have done more changes in the RFC that needs to be reverted.

Reply:
My bad. Fixed.